{"filter":false,"title":"exception_messages.py","tooltip":"/ib_miniprojects_backend/user_authentication/constants/exception_messages.py","ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":11,"column":1},"end":{"row":11,"column":1},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"hash":"83a5a1e2f41295f91d06413f0598ef523e7aa2e8","undoManager":{"mark":0,"position":0,"stack":[[{"start":{"row":0,"column":0},"end":{"row":11,"column":1},"action":"insert","lines":["INVALID_USER_NAME = (","    \"Invalid username, try with valid username\",","    \"INVALID_USER_NAME\"",")","INVALID_PASSWORD = (","    \"Invalid password, try with valid password\",","    \"INVALID_PASSWORD\"",")","INVALID_USER_ID = (","    \"Invalid user id, try with valid user id\",","    \"INVALID_USER_ID\"",")"],"id":1}]]},"timestamp":1593442718384}